### PR TITLE
(#22) Add health-check:health command

### PR DESCRIPTION
**Pull request by nick322**
Because [#44](https://github.com/ans-group/laravel-health-check/pull/44) hasn't been updated for a long time...
So I have reference to  [#44](https://github.com/ans-group/laravel-health-check/pull/44) to fix the conflict and some modifications
Hope this feature can be updated

### CODE REVIEW

**Comment by Gman98ish** [comment]
Thanks for this @nick322 I'll take a look when I get a sec

**Comment by phily245** [comment]
@nick322 I'm now on the case

File: src/Commands/StatusCommand.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            if ($status->isProblem()) {
+                $problems[] = [$check->name(), $status->name(), $status->message()];
+            }
+        }
+
+        if (!$isOkay = empty($problems)) {
+            $this->table(['name', 'status', 'message'], $problems);
+        }
+
+        return $isOkay ? 0 : 1;
```
Comment (phily245) [inline src/Commands/StatusCommand.php:52]: The only change I'd suggest here is adding a success message, as when I used the command I was left wondering if it had broken or it wasn't doing anything. Maybe something like:

```suggestion
        $this->info('All checks passed successfully');

        return $isOkay ? 0 : 1;
```

**Review by phily245 — CHANGES_REQUESTED** [review]
Some nice refactoring added in here too, I have added a suggestions for visual feedback on success

File: src/Controllers/HealthCheckController.php [hunk 2]
```diff
@@ -2,46 +2,41 @@
...
+            }
+
+            if ($status->getStatus() == Status::DEGRADED && $hasProblem == false) {
+                Arr::set($body, 'status', Status::DEGRADED);
             }
         }
 
-        return new Response($body, $isProblem ? 500 : 200);
+        return response()
+            ->json($body, in_array(Arr::get($body, 'status'), [Status::DEGRADED, Status::OKAY])  ? 200 : 500);
```
Comment (phily245) [inline src/Controllers/HealthCheckController.php:40]: I have just merged another PR which changes this line, here's how it should look now:

```suggestion
            ->json($body, in_array(Arr::get($body, 'status'), [Status::DEGRADED, Status::OKAY])  ? 200 : config('healthcheck.default-problem-http-code', 500));
```

**Review by phily245 — APPROVED** [review]
Thank you for contributing @nick322!

**Pull request merged**